modules/apps/transfer/keeper/keeper.go

```diff
@@ -146,28 +146,28 @@ func (k Keeper) IterateDenomTraces(ctx sdk.Context, cb func(denomTrace types.Den
 
 // GetTotalEscrowForDenom gets the total amount of source chain tokens that
 // are in escrow, keyed by the denomination.
-func (k Keeper) GetTotalEscrowForDenom(ctx sdk.Context, denom string) math.Int {
+func (k Keeper) GetTotalEscrowForDenom(ctx sdk.Context, denom string) sdk.Coin {
 	store := ctx.KVStore(k.storeKey)
 	bz := store.Get(types.TotalEscrowForDenomKey(denom))
 	if bz == nil {
-		return math.ZeroInt()
+		return sdk.NewCoin(denom, sdk.ZeroInt())
```

Comment:
do we want to return an empty coin literal here or keep this? 

personally I think either is fine

Reply:
empty coin literal I think would be my preference (this should mean that the key isn't set